File: backend/src/main/java/com/project/dto/appointment/response/AppointmentResponseDTO.java
package com.project.dto.appointment.response;

import java.time.LocalDateTime;

import com.project.enums.AppointmentStatus;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;



@Data
@AllArgsConstructor
public class AppointmentResponseDTO {
    public AppointmentResponseDTO() {
	}
	private Long appointmentId;
    private Long userId;
    private String professionalName;
    private LocalDateTime startTime;
    private LocalDateTime endTime;
    private AppointmentStatus status;
}

File: backend_dotnet/SafeMind.Core/DTOs/Appointment/Response/AppointmentResponseDTO.cs
namespace SafeMind.Core.DTOs.Appointment.Response
{
    public class AppointmentResponseDTO
    {
        public long AppointmentId { get; set; }
        public long UserId { get; set; }
        public string? FullName { get; set; }
        public string? ProfessionalName { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        // Use string for status to avoid missing enum dependency
        public string Status { get; set; } = string.Empty;
    }
}
